index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
I am not sure what is best. One is relative to the device orientation itself, and one is relative to the earth.

It seems better to have two separate objects for that to avoid confusion.

Now as both MS and Android calls the one relative to earth for Device Orientation Sensor, we should probably keep that name, but both could potentially inherit from one Orientation Sensor.

I think that makes sense, as we may have other devices in the future relative to other things. Like with "device" we usually mean the device where the code is running. With an external headset, that might not be the device actually running the code, so I could imagine a 

GamepadSensor extends Relative Orientation Sensor
HeadsetSensor extends Orientation Sensor.

Actually looking at the above, I now see that the name "device" is probably quite bad, because it is not clear what it refers to, so just calling it Orientation Sensor may make sense.

I guess we could have

* Orientation Sensor (relative to the world / earth)
* Relative Orientation Sensor (relative to the device itself) extends the above.

Then in the future we might have external sensors inheriting those.